exporter/kafkaexporter/config.go

```diff
@@ -67,6 +68,14 @@ type Metadata struct {
 type Producer struct {
 	// Maximum message bytes the producer will accept to produce.
 	MaxMessageBytes int `mapstructure:"max_message_bytes"`
+
+	// RequiredAcks Number of acknowledgements required to assume that a message has been sent.
+	// https://pkg.go.dev/github.com/Shopify/sarama@v1.30.0#RequiredAcks
+	// The options are:
+	//   0 -> NoResponse.  doesn't send any response
+	//   1 -> WaitForLocal. waits for only the local commit to succeed before responding ( default )
+	//   -1 -> WaitForAll. waits for all in-sync replicas to commit before responding.
+	RequiredAcks sarama.RequiredAcks `mapstructure:"required_acks"`
```

Comment:
Please add validation that this value is in the enumeration to the `Validate()` method below.

Reply:
I have added validation to check that `RequiredAcks` is correctly formatted